packages/victory-vendor/package.json

```diff
@@ -27,6 +27,13 @@
     "build": "node ./scripts/build.js"
   },
   "dependencies": {
+    "@types/d3-array": "^3.0.3",
+    "@types/d3-ease": "^3.0.0",
```

Comment:
I added these as `dependencies` ... does that make sense?  Maybe dev deps makes more sense?

Reply:
Hmm... let me think about this...

Reply:
I think `dependencies` if our upstream consumers need these types to actually be present.

Reply:
This makes sense to me!